packages/itwinui-react/src/core/RadioTiles/RadioTile.tsx

```diff
@@ -28,29 +57,82 @@ type RadioTileProps = {
  * <RadioTile label='My tile' description='Some info' icon={<SvgSmileyHappy />} />
  */
 export const RadioTile = React.forwardRef((props, ref) => {
-  const { icon, label, description, className, style, ...rest } = props;
+  const {
+    icon,
+    label,
+    style,
+    description,
+    setFocus = false,
+    tileProps = {},
+    inputProps = {},
+    contentProps = {},
+    iconProps = {},
+    labelProps = {},
+    subLabelProps = {},
+  } = props;
 
   const inputElementRef = React.useRef<HTMLInputElement>(null);
   const refs = useMergedRefs<HTMLInputElement>(inputElementRef, ref);
 
+  React.useEffect(() => {
+    if (inputElementRef.current && setFocus) {
+      inputElementRef.current.focus();
+    }
+  }, [setFocus]);
+
+  const { className: tileClassName, ...restTileProps } = tileProps;
+  const { className: inputClassName, ...restInputProps } = inputProps;
+  const { className: contentClassName, ...restContentProps } = contentProps;
+  const { className: iconClassName, ...restIconProps } = iconProps;
+  const { className: labelClassName, ...restLabelProps } = labelProps;
+  const { className: subLabelClassName, ...restSubLabelProps } = subLabelProps;
+
   return (
-    <Box as='label' className={cx('iui-radio-tile', className)} style={style}>
+    <Box
+      as='label'
+      className={cx('iui-radio-tile', tileClassName)}
+      style={style}
+      {...restTileProps}
```

Comment:
i think `tileProps` is unnecessary. we should forward the top-level `...rest` props here, and use the top-level `className` as well.

also i believe we should use the forwarded ref here, rather than on the input.

also pinging @gretanausedaite for a second opinion

Reply:
https://github.com/iTwin/iTwinUI/pull/1409/files#diff-c1ed339f177f865b3908b978bff96791d3171cbe4bb8f0ddbca5cd73ca77d46aL31

They were defined in this line when setting the props

Reply:
Added it back, now it looks like this: 
![image](https://github.com/iTwin/iTwinUI/assets/44583237/fce77c77-89ca-432e-af2c-1c89d2123d1c)

Reply:
Fixed!

Reply:
lines 65-66 can also be removed. because we are not doing anything special with it, we don't need to create or combine refs, we can simply pass the forwarded ref to the label element

Reply:
Done!

Reply:
looks good. i'll leave this comment open for @gretanausedaite to verify

Reply:
Looks good to me.